wallet/reference/multichain-api.md

```diff
@@ -0,0 +1,396 @@
+
+:::note See also
+- [About the Multichain API](../concepts/multichain-api.md)
+- [Interact with multiple networks](../how-to/manage-networks/use-multichain.md)
+:::
+
+## Methods
+
+### `wallet_createSession`
+
+Creates a multichain connection with a wallet, authorizing that wallet with the specified set of scopes
+and properties.
+This method is defined in [CAIP-25](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-25.md).
+
+#### Parameters
+
+- `optionalScopes`: `object` - (Optional) [CAIP-217](https://chainagnostic.org/CAIPs/caip-217) authorization scopes the wallet can support in order to be used with this dapp.
+  If scopes are specified, only the following properties are supported:
+  - `references`: `array` - (Optional) A list of references to specific blockchains for the namespace of this scope.
+    This property is mainly used when there would otherwise be duplicate scopes.
+  - `methods`: `array` - A list of JSON-RPC methods the wallet must support in order to be used with this dapp.
+  - `notifications`: `array` - A list of JSON-RPC notifications the wallet must support in order to be used with this dapp.
+  - `accounts`: `array` - (Optional) A list of [CAIP-10](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-10.md)
+    account IDs valid within this scope.
+    Dapps should only supply this property when they know the set of accounts they would like the user to permit.
+    When supplied, these accounts are preselected by default in the account selection process.
+    Dapps typically omit this property for the user to select their own accounts.
+- `sessionProperties`: `object` - (Optional) Properties that the wallet can use to determine if the connection is valid.
+- `requiredScopes`: `object` - (Optional) [CAIP-217](https://chainagnostic.org/CAIPs/caip-217) authorization scopes the wallet must support in order to be used
+  with this dapp.
+  We don't recommend using `requiredScopes` with MetaMask.
+
+#### Returns
+
+The scopes and properties of the created connection.
+
+#### Example
+
+<Tabs>
+<TabItem value="Request">
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "wallet_createSession",
+  "params": {
+    "optionalScopes": {
+      "eip155": {
+        "references": ["1", "137"],
+        "methods": ["eth_sendTransaction", "eth_signTransaction", "eth_sign", "get_balance", "personal_sign"],
+        "notifications": ["accountsChanged", "chainChanged"]
+      },
+      "eip155:10": {
+        "methods": ["get_balance"],
+        "notifications": ["accountsChanged", "chainChanged"]
+      },
+      "eip155:0": {
+        "methods": ["wallet_getPermissions", "wallet_creds_store", "wallet_creds_verify", "wallet_creds_issue", "wallet_creds_present"],
+        "notifications": []
+      },
+      "eip155:42161": {
+        "methods": ["eth_sendTransaction", "eth_signTransaction", "get_balance", "personal_sign"],
+        "notifications": ["accountsChanged", "chainChanged"]
```

Comment:
there are no `accountsChanged` or `chainChanged` notifications for EVM in the Multichain API

Reply:
same thing for the result.

`eth_subscription` is the only valid notification for evm currently

Reply:
updated examples